cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -21,24 +25,64 @@ type InstallerIntegrationTestSuite struct {
 	tagsuite.Suite
 }
 
+func (suite *InstallerIntegrationTestSuite) TestInstallFromInstallationError() {
+	suite.OnlyRunForTags(tagsuite.Installer)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnCmdWithOpts(
+		ts.InstallerExe,
+		e2e.WithArgs(filepath.Join(ts.Dirs.Work, "installation"), "--source-path", ts.Dirs.Base),
+		e2e.AppendEnv(constants.DisableUpdates+"=false"))
+
+	cp.Expect("Cannot run state-installer from an installation directory.")
+	cp.ExpectExitCode(1)
+}
+
 func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 	suite.OnlyRunForTags(tagsuite.Installer, tagsuite.Critical)
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
+	// Determine URL of installer archive.
+	baseUrl := "https://state-tool.s3.amazonaws.com/update/state"
+	archiveExt := "tar.gz"
+	if runtime.GOOS == "windows" {
+		archiveExt = "zip"
+	}
+	archiveUrl := fmt.Sprintf("%s/%s/%s/%s-amd64/state-%s-amd64-%s.%s",
+		baseUrl,
+		constants.BranchName,
+		constants.Version,
+		runtime.GOOS,
+		runtime.GOOS,
+		constants.Version,
+		archiveExt)
+
+	// Fetch it.
+	b, err := download.GetDirect(archiveUrl)
+	suite.Require().NoError(err)
+
+	// Extract it.
+	installerDir := filepath.Join(ts.Dirs.Work, "installer")
+	if runtime.GOOS != "windows" {
+		suite.Require().NoError(unarchiver.NewTarGzBlob(b).Unarchive(installerDir))
+	} else {
+		suite.Require().NoError(unarchiver.NewZipBlob(b).Unarchive(installerDir))
+	}
```

Comment:
This test is specifically testing the local source (note the suffix: `FromLocalSource()`).

So this is breaking the test.